src/tools/sssctl/sssctl_config.c

```diff
@@ -34,6 +34,37 @@
 
 
 #ifdef HAVE_LIBINI_CONFIG_V1_3
+
+static char *sssctl_config_snippet_path(TALLOC_CTX *ctx, const char *path)
+{
+    char *tmp;
+    const char *snippet_dir;
+    const char delimiter = '/';
+    char *dpos;
+
+    tmp = talloc_strdup(ctx, path);
+    if (!tmp) {
```

Comment:
Maybe some error message here?

Reply:
Message is actually provided in caller

    if (config_snippet_path == NULL) {
        DEBUG(SSSDBG_CRIT_FAILURE, "Unable to create snippet path\n");
        ret = ENOMEM;